addons/web_editor/static/lib/odoo-editor/test/utils.js

```diff
@@ -354,6 +365,9 @@ export async function testEditor(Editor = OdooEditor, spec, options = {}) {
             customErrorMessage('contentAfter', value, spec.contentAfter));
     }
     await testNode.remove();
+    if (nextTestIsMobile) {
+        await testEditor(Editor, spec, options, isMobile=true);
```

Comment:
Note that (unfortunately) `isMobile = true` does not do what you think it does in JS. It does result into `true`, which is why it works, but it does _not_ match the paremeters properly based on their name like Python does. Therefore, I would prefer to remove it because it gives a false impression of safety.

What does work however, is using an options object like you'd do a Python dict. I see that we already have one for this function so we can use it like this:
```js
await testEditor(Editor, spec, { ...options, isMobile: true });
```